cni/plugin.go

```diff
@@ -158,9 +158,6 @@ func (plugin *Plugin) InitializeKeyValueStore(config *common.PluginConfig) error
 			log.Printf("[cni] Failed to create store: %v.", err)
 			return err
 		}
-
-		// Force unlock the json store if the lock file is left on the node after reboot
-		removeLockFileAfterReboot(plugin)
```

Comment:
can you remove the func definition for this as we don't need this anymore? You can remove it from linux as well. ( the one in linux is just printing the reboot time )

func removeLockFileAfterReboot(plugin *Plugin) {
	rebootTime, _ := platform.GetLastRebootTime()
	log.Printf("[cni] reboot time %v", rebootTime)
}

Reply:
> xref: #774 this should resolve that issue partially

not sure if this fixes that issue.

Reply:
lets have it for a while and will remove this once we move to filelock api